scripts/quarantine.yaml

```diff
@@ -7,7 +7,9 @@
 #  platforms:
 #    - None
 
+# tests not passing while using zephyr-sdk=0.14.1
```

Comment:
Please add this as a `comment` entry, like in this example:
https://docs.zephyrproject.org/latest/develop/test/twister.html#quarantine
Then the info will be propagated to test reports

Reply:
done